File: concepts/SOLID/PHP/5-DIP/a/src/taxes/FullTimeTaxCalculator.php
<?php 

namespace app\taxes;

use app\personnel\Employee;

class FullTimeTaxCalculator implements TaxCalculator
{

  private const RETIREMENT_TAX_PERCENTAGE = 10;
  private const INCOME_TAX_PERCENTAGE = 16;
  private const BASE_HEALTH_INSURANCE = 100;

  public function calculate(Employee $employee)
  {    
    $monthlyIncome = $employee->getMonthlyIncome();
    return self::BASE_HEALTH_INSURANCE + ($monthlyIncome * self::RETIREMENT_TAX_PERCENTAGE)/ 100 +
    ($monthlyIncome * self::INCOME_TAX_PERCENTAGE) / 100 ;
  }
}

File: concepts/SOLID/Java/2-OCP/GlobomanticsHR/src/main/java/hr/taxes/FullTimeTaxCalculator.java
package hr.taxes;

import hr.personnel.Employee;

public class FullTimeTaxCalculator implements TaxCalculator {
    private final int RETIREMENT_TAX_PERCENTAGE = 10;
    private final int INCOME_TAX_PERCENTAGE = 16;
    private final int BASE_HEALTH_INSURANCE = 100;

    public double calculate(Employee employee) {
        int monthlyIncome = employee.getMonthlyIncome();
        return BASE_HEALTH_INSURANCE +
                (monthlyIncome * RETIREMENT_TAX_PERCENTAGE) / 100 +
                (monthlyIncome * INCOME_TAX_PERCENTAGE) / 100;
    }
}
